NebulaPatcher/NebulaPlugin.cs

```diff
@@ -43,6 +44,14 @@ private void InitPatches()
             try
             {
                 var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+                //Environment.SetEnvironmentVariable("MONOMOD_DMD_TYPE", "mb");
+                //Environment.SetEnvironmentVariable("MONOMOD_DMD_DUMP", "./mmdump");
+                /*
+                foreach (FileInfo file in new DirectoryInfo("./mmdump").GetFiles())
+                {
+                    file.Delete();
+                }
+                */
```

Comment:
Maybe we could add another build target that enables this

Reply:
Im not sure how this would be done :/

Reply:
I'll look at doing this later